### ISSUE TITLE
FreeBSD

### ISSUE THREAD

**Issue by SylvainCorlay** [conda-forge/miniforge#17]
What would it take to bootstrap a new mini[forge/conda] for FreeBSD as they are no packages whatsoever.

I would be willing to help, although it would not be funded / dayjob work for me. I have this FreeBSD server hanging arround and it has been itching me to do some data science on it for some time.

**Comment by isuruf** [comment]
You'll have to first build conda-build and its dependencies on a normal python installation. Then modify `conda-build` (and maybe `conda`) to make `freebsd-*` take the same code paths as `linux-*`.

**Comment by SylvainCorlay** [comment]
Thanks!
I should give it a try.

**Comment by isuruf** [comment]
You could also cross-compile from linux-64 to freebsd using clang. This should make it easier as you'd already have a conda installation, compiler, linker, archiver, etc.
I checked conda and conda-build, and you need to change a handful of lines.

**Comment by isuruf** [comment]
Also anaconda.org doesn't like os names other than `linux`, `osx`, `win`.

**Comment by scopatz** [comment]
Yeah, but we can't be bound by what anaconda.org has.  It is pretty old

**Comment by isuruf** [comment]
I'm just mentioning the issues we'll run into.

**Comment by isuruf** [comment]
We can cross compile even without access to the hardware. Closing as there's nothing to do in miniforge until somebody adds support to conda, conda-build and starts a migration. A better place would be conda-forge/conda-forge.github.io
